### PR TITLE
Updated sensors for Stulz WIB and updated MIB

### PR DESCRIPTION
**Pull request by martinberg**
Note that SNMP retries and timeout had to be adjusted in the settings for the device. Otherwise be prepared for timeouts

image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by jepke** [comment]
@martinberg you're PR is likely going to break my system :) which is likely the reason tests fail. Little hard to go through test logs on my phone but might be the sysObjectID that is altered in https://github.com/librenms/librenms/pull/16072/files#diff-e3022efe8e719c2d9f039a0c04859c73eef1cb265842860f2e1a5ac39ff63fb6R10

Ill try to pull your PR tomorrow that'll give me a better view.

**Comment by martinberg** [comment]
> @martinberg you're PR is likely going to break my system :) which is likely the reason tests fail. Little hard to go through test logs on my phone but might be the sysObjectID that is altered in https://github.com/librenms/librenms/pull/16072/files#diff-e3022efe8e719c2d9f039a0c04859c73eef1cb265842860f2e1a5ac39ff63fb6R10
> 
> Ill try to pull your PR tomorrow that'll give me a better view.

Hello!
Have you been able to test my changes?
Hopefully I wont break too much of your system :)

It is possible the sysObjectID is the cause but to me it seems like it dislike the field hardware, version and etc that has been added.

The only error I can understand is the following, but maybe it can be accepted anyway? :)
`[modules.sensors.waterflow.data[1].multiplier] Double value found, but an integer is required`

File: includes/definitions/stulz.yaml [hunk 1]
```diff
@@ -4,6 +4,46 @@ type: environment
 icon: stulz
 over:
     - { graph: device_bits, text: 'Device Traffic' }
+snmp_bulk: true
 discovery:
     - sysObjectID:
-        - .1.3.6.1.4.1.29462.
+        - .1.3.6.1.4.1.29462.10
```
Comment (murrant) [inline includes/definitions/stulz.yaml:10]: The oid you removed matches the one you added and yo likely removed support for other devices.

File: includes/definitions/stulz.yaml [hunk 2]
```diff
@@ -4,6 +4,46 @@ type: environment
...
+discovery_modules:
```
Comment (murrant) [inline includes/definitions/stulz.yaml:11]: Do not set modules in definitions, that is up to the user.

File: includes/definitions/discovery/stulz.yaml [hunk 3]
```diff
@@ -8,177 +8,252 @@ modules:
...
+                    oid: Stulz-WIB8000-MIB::unitReturnAirTemperature
+                    value: Stulz-WIB8000-MIB::unitReturnAirTemperature
                     num_oid: '.1.3.6.1.4.1.29462.10.2.1.1.1.1.1.1.1.1192.{{ $index }}'
+                    descr: 'Unit {{ $subindex1 }} Return Air Temperature'
                     index: 'unitReturnAirTemperature.{{ $index }}'
-                    descr: 'Unit {{ $index }} return temp'
                     divisor: 10
+                    group: Return Air Temperature
                 -
-                    oid: unitWaterInTemperature1
```
Comment (murrant) [inline includes/definitions/discovery/stulz.yaml:111]: Make sure not to remove support for other devices.  Discovery will determine which ones are available for polling automatically.

**Pull request closed without merging**